bin/ch/WScriptJsrt.cpp

```diff
@@ -34,6 +34,25 @@
 #endif // FreeBSD or unix ?
 #endif // _WIN32 ?
 
+
+struct ThreadLocalData
+{
+    ThreadLocalData()
+        :threadData(nullptr)
+    {
+    }
+    ~ThreadLocalData()
+    {
+        if (threadData)
+        {
+            delete threadData;
```

Comment:
sure, will do.

btw, just notice semaphore is not implemented in our PAL, is it hard to wrap linux semaphore to be like the Win32 one?